KokoDajMu/KokoDajMu_MAIN/Constants/Constants.cs

```diff
@@ -8,5 +8,9 @@ namespace KokoDajMu
 {
     public class Constants
     {
+        public const int CONVERT_TO_MINUTES = 60;
+        public const int CONVERT_TO_HOURS = 3600;
+        public const string PATH_TO_TEXT_FILES_TO_READ = @"\\files\Public\Transfer\Kaloian Karaivanov\Internship\internship_tasks\KokoDajMu\KokoDajMu_MAIN\TextDocuments\KokoDajMu.txt";
```

Comment:
Path should be something like \..\TextDocuments\KokoDajMu.txt so that it is not specific to your machine